includes/admin/class-sensei-learners-main.php

```diff
@@ -471,18 +472,24 @@ protected function get_row_data( $item ) {
 				 * This filter runs on the learner management screen for a specific course.
 				 * It provides the learner row column details.
 				 *
-				 * @param array $columns{
-				 *   type string $title
-				 *   type string $date_started
-				 *   type string $course_status (completed, started etc)
-				 *   type html $action_buttons
+				 * @param string tag
```

Comment:
We don't need to add this as we only need to describe the parameters that will be passed when someone calls `add_filter` for this hook.